components/RecipeCard.js

```diff
@@ -61,43 +117,38 @@ export default function RecipeCard({
 }) {
   return (
     <StyledArticle>
+      <StyledContentWrapper>
+        <ReadMoreLink href={`/${recipe._id}`}>
+          <StyledInfoBox>
+            <StyledHeader>{recipe.title}</StyledHeader>
+            <StyleItemsList>
+              {recipe.symptoms.map((symptom) => (
+                <StyledItems key={symptom}>{symptom}</StyledItems>
+              ))}
+            </StyleItemsList>
+          </StyledInfoBox>
+          <AuthorBox>
+            <StyledRecipeBy>Recipe by</StyledRecipeBy>
+            <StyledHerbie>Herbie</StyledHerbie>
+          </AuthorBox>
+        </ReadMoreLink>
+      </StyledContentWrapper>
+
       <StyledImageWrapper>
-        <Image
+        <StyledImage
           src={recipe.image}
-          style={{ width: "100%" }}
           height={200}
           width={200}
-          objectFit="contain"
           alt={recipe.title}
-        ></Image>
+        ></StyledImage>
```

Comment:
always use self closing tags if possible. Please check your codebase and update it accordingly.